recipes/coverage/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "3.7.1" %}
+{% set md5 = "c47b36ceb17eaff3ecfab3bcd347d0df" %}
+{% set pkgname = "coverage" %}
+
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pkgname[0] }}/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
```

Comment:
So we need to add some more stuff to them. Hence why we broke them out. I'll add another PR for the changes @ericmjl.